clair/api.go

```diff
@@ -78,6 +79,34 @@ func (a *apiV1) pushLayer(layer *layer) error {
 	return nil
 }
 
+const maxRetries = 5
```

Comment:
I think there is a 5 min timeout in the kubei so we need to avoid it.
Maybe we should pass the retries in an env var to the klar job?